provisioning_client/tests/hsm_client_riot_ut/hsm_client_riot_ut.c

```diff
@@ -175,17 +196,24 @@ static char* umocktypes_stringify_RIOT_ECC_PUBLIC(const RIOT_ECC_PUBLIC* value)
     }
     else
     {
-        int length = snprintf(NULL, 0, "{ %p, %p, %d }",
-            value->x.data, value->y.data, (int)value->infinity);
+        int length = snprintf(NULL, 0, "{ %d, %zd, %p }",
+            value->X.s, value->X.n, value->X.p);
+        length += snprintf(NULL, 0, "{ %d, %zd, %p }",
+            value->Y.s, value->Y.n, value->Y.p);
+        length += snprintf(NULL, 0, "{ %d, %zd, %p }",
+            value->Z.s, value->Z.n, value->Z.p);
         if (length < 0)
         {
             result = NULL;
         }
         else
         {
-            result = (char*)my_gballoc_malloc(length + 1);
-            (void)snprintf(result, length + 1, "{ %p, %p, %d }",
-                value->x.data, value->y.data, (int)value->infinity);
+            length++;
```

Comment:
Assuming this is for the `NULL` character have a comment stating it. I had to double check around to make sure that's why it was here.

Reply:
Oh good catch, yeah it took me a little bit to realize why length+1 in the first place, should have commented on that.

Reply:
Added comment